### PR TITLE
Add bias correction feature

### PR DESCRIPTION
**Pull request by caibengbu**
This pull request add the bias correction feature to the GLFixedEffectModels package. It returns the same numerical results with `alpaca::BiasCorr()` in R. 

TO-DO:

- [x] allow the binwidth to be different than 0.
- [ ] do more robustness test with missing values etc.

### CODE REVIEW

**Comment by jmboehm** [comment]
Great! Could you also add the following:
- hardcoded values that you compare to the estimates with `@test` (see e.g. `test/nlreg.jl`)
- a test with probit link?

**Comment by caibengbu** [comment]
Thank you, Johannes. I updated the test script. The two-way Logit test was fine but the coefficient didn't match for models that use a Probit link.

I use `RCall` to get the result from `alpaca::feglm` and compare the result with our function `nlreg`. They appear to be different before bias correction with `atol = 1e-4`

**Comment by jmboehm** [comment]
Thank you. That's concerning-- yes, it would be great if you could look into what's causing the difference in the probit regressions (I should have really added these to the tests in the first place...). But that can go into a different PR, so feel free to comment out the probit test stuff for now.

By the way, to have biascorr_test.jl also run by the Github CI, you also need to add the file in `test/runtests.jl`. If you want to use some other dependencies in that script that are not already listed in `Project.toml` (like `RCall`), you'll have to add it to the test vector, otherwise the tests will fail.
```
[targets]
test = ["Test", "Random", "Statistics", "RDatasets", "StableRNGs"]
```
Feel free to message me on Slack if the tests keep failing.

File: Project.toml [hunk 1]
```diff
@@ -47,4 +47,4 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```
Comment (jmboehm) [inline Project.toml:48]: add to [extras]
```
RCall = "6f49c342-dc21-5d91-9882-a32aef131414"
```

**Comment by jmboehm** [comment]
Awesome, thank you!

**Pull request merged**